lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -303,6 +310,23 @@ private Optional<InheritedIdentityJwtClaim> getInheritedIdentityClaim(JWTClaimsS
         }
     }
 
+    private Optional<ListOfStringValues> getGvcsAccessTokenClaim(JWTClaimsSet claimsSet)
```

Comment:
Should we commonise this with inherited identity, e.g. `getStringListClaim(JWTClaimsSet claimsSet)`?

Reply:
done